docker/Dockerfile

```diff
@@ -96,26 +80,16 @@ RUN apt-get update && \
         rsync \
         systemd \
         systemd-container \
+        systemd-resolved \
         u-boot-tools \
         unzip \
         user-mode-linux \
         xfsprogs \
         xz-utils \
-        zip && \
-    rm -rf /var/lib/apt/lists/*
-
-# Enable backports for the Arch dependencies
-RUN echo "deb http://ftp.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list
-
-# NOTE: Explicitly install arch-install-scripts from backports. The normal one
-# lacks pactrap.
-# Install Arch dependencies
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
+        zip \
         makepkg \
-        pacman-package-manager && \
-    apt-get install -y --no-install-recommends \
-        -t bullseye-backports \
+        pacman-package-manager \
```

Comment:
as a follow-up note we should remove the arch packages from the debian-based images and build arch-based images separately (#445).